linux_docker_resources/Dockerfile

```diff
@@ -3,6 +3,7 @@ ARG BRIDGE=false
 ARG INSTALL_CONNEXT_DEBS=false
 ARG PLATFORM=x86
 ARG ROS1_DISTRO=noetic
+ARG ROS_DISTRO=rolling
```

Comment:
Although this build argument is not yet used, and is not strictly necessary for this patch, I felt it worthwhile to apply the argument to both Ubuntu and RHEL docker builds. If preferred I could walk this back though.

Reply:
Agreed, I think we should keep them consistent.  If we also go forth with the plan to make these Dockerfiles install "everything needed to build", then I think we'll need this anyway.